dist/jsoneditor.js

```diff
@@ -1247,23 +1247,6 @@ JSONEditor.Validator = Class.extend({
           });
         }
       }
-
-      // `uniqueItems`
-      if(schema.uniqueItems) {
```

Comment:
In this branch can you revert the merge done https://github.com/HULFT-Inc/json-editor/commit/765914b58837b6a05c68d22d903a20b02d9e29b7
I don't think we are using this build, so want to make sure this fix doesn't get included here.